test/schemas/actionParamsValidation/Person.schema

```diff
@@ -3,7 +3,7 @@
   FullName: { category: 'FullName' },
   Mother: { category: 'Person' },
   ChangeDOB: Action({
-    Execute: (session, ctx, args, callback) => callback(null, 'ok'),
+    Execute: async (session, ctx, args, callback) => 'ok',
```

Comment:
You should remove the `callback` argument here. This also applies to other `Action`s.
```suggestion
    Execute: async (session, ctx, args) => 'ok',
```